src/js/utils.js

```diff
@@ -20,7 +44,8 @@ this.exports = {
             {
                 "type": "setHold",
                 "params": {
-                  "holdType": "indefinite", 
+                  "holdType": this.calculateHoldType(thermostat.settings), 
```

Comment:
I think we should just totally drop the holdType field. It's marked as not required in ecobee API docs. And if I'm not wrong, if you skip the field, hold is created according to default hold type configuration on tstats. Less code for us.